explore/src/main/scala/explore/targeteditor/SelectButtonEnumView.scala

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.ui.primereact
+
+import cats.syntax.all.*
+import eu.timepit.refined.types.string.NonEmptyString
+import japgolly.scalajs.react.*
+import japgolly.scalajs.react.vdom.html_<^.*
+import lucuma.core.util.Display
+import lucuma.core.util.Enumerated
+import react.common.*
+import react.primereact.SelectButton
+import react.primereact.SelectItem
+
+import scalajs.js
+
+case class SelectButtonEnumView[V[_], A](
```

Comment:
We should move this to lucuma-ui eventually

Reply:
Yes, definitely.